Testing/functional/tests/cypress/integration/ui/covid19/publicVaccineCard.js

```diff
@@ -45,7 +45,16 @@ function clickVaccineCardEnterButton() {
 
 describe("Public Vaccine Card Form", () => {
     beforeEach(() => {
-        cy.enableModules(["Immunization"]);
+        cy.configureSettings(
+            {
+                covid19: {
+                    publicCovid19: {
+                        showFederalProofOfVaccination: true,
+                    },
+                },
+            },
+            ["Immunization"]
+        );
```

Comment:
I don't think showFederalProofOfVaccination needs to be set for this test.

Reply:
Fixed in Fixed in AB#14995